### PR TITLE
add the new layer for `rapid-mapillary-features`

### PR DESCRIPTION
**Pull request by MahmoudMuhammedAli**
- Create the new renderer for the `rapid_features` layer. 
- style it differently to stand out from the current `mapillary_map_features`
- adjust the tests to make room for the newly added node.

### CODE REVIEW

File: modules/renderer/photos.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import { utilRebind } from '../util/rebind';
 
 export function rendererPhotos(context) {
     var dispatch = d3_dispatch('change');
-    var _layerIDs = ['streetside', 'mapillary', 'mapillary-map-features', 'mapillary-signs', 'openstreetcam'];
+    var _layerIDs = ['streetside', 'mapillary', 'mapillary-map-features', 'mapillary-signs', 'openstreetcam', 'rapid-features'];
```
Comment (Bonkles) [inline modules/renderer/photos.js]: `rapid-features` is a little generic for the layer name. how about `rapid-mapillary-features`?

File: package.json [hunk 2]
```diff
@@ -32,6 +32,7 @@
     "dist:svg:maki": "svg-sprite --symbol --symbol-dest . --shape-id-generator \"maki-%s\" --symbol-sprite dist/img/maki-sprite.svg node_modules/@mapbox/maki/icons/*.svg",
     "dist:svg:mapillary:signs": "svg-sprite --symbol --symbol-dest . --symbol-sprite dist/img/mapillary-sprite.svg node_modules/mapillary_sprite_source/package_signs/*.svg",
     "dist:svg:mapillary:objects": "svg-sprite --symbol --symbol-dest . --symbol-sprite dist/img/mapillary-object-sprite.svg node_modules/mapillary_sprite_source/package_objects/*.svg",
+    "dist:svg:rapid:objects": "svg-sprite --symbol --symbol-dest . --symbol-sprite dist/img/rapid-object-sprite.svg node_modules/mapillary_sprite_source/package_objects/*.svg",
```
Comment (Bonkles) [inline package.json]: We discussed this already, but I'll flag this: we're creating a whole new file here when we really only need 4 of the sprites. @hieule21 feel free to respond with the task # of the tech debt task we talked about yesterday!

**Review by Bonkles — CHANGES_REQUESTED** [review]
just one super quick change request to name the new layer something more specific.

File: package.json (hunk 2 above)
Comment (MahmoudMuhammedAli) [reply package.json:35]: changed `rapid-features` to `rapid-mapillary-features` and updated the affected files.

Comment (hieule21) [reply package.json]: We discussed this further on Discord and decided to go this way.

File: modules/renderer/photos.js (hunk 1 above)
Comment (hieule21) [reply modules/renderer/photos.js]: Mahmoud, let me take a close look at this to make sure the name is consistent everywhere else in the code.

**Pull request merged**